### ISSUE TITLE
Indicator VCV = S?

### ISSUE THREAD

**Issue by M-E-Rademaker** [M-E-Rademaker/cSEM#119]
Currently we have  `Indicator_VCV = `S` in the final cSEMResults output. But note that S is computed by `polycor::hetcor()`. I dont think this is a problem, however, we should be aware that `S` is no longer always equal to Pearson correlation aka. `cor(data)`. 
For example: To compute the SRMR both matrices need to be correlation matrices right?

**Comment by FloSchuberth** [comment]
Yes, this is true, but I see no problem using the polychoric/polyserial correlation instead of the Pearson correlation. Indeed, it often recommended to use the polychoric correlation instead of the Pearson for post estimation statistics, see my book chapter with Gabriele Cantaluppi for some examples such as Cronbach's alpha.  It might be different for the robust correlation.
